ZBaselines/Common/EntryPoints/core_ep-list.tsv

```diff
@@ -3,6 +3,7 @@ Data.DataViewReference	Pass dataview from memory to experiment	Microsoft.ML.Runt
 Data.IDataViewArrayConverter	Create and array variable	Microsoft.ML.Runtime.EntryPoints.CrossValidationBinaryMacro	MakeArray	Microsoft.ML.Runtime.EntryPoints.CrossValidationBinaryMacro+ArrayIDataViewInput	Microsoft.ML.Runtime.EntryPoints.CrossValidationBinaryMacro+ArrayIDataViewOutput
 Data.PredictorModelArrayConverter	Create and array variable	Microsoft.ML.Runtime.EntryPoints.CrossValidationBinaryMacro	MakeArray	Microsoft.ML.Runtime.EntryPoints.CrossValidationBinaryMacro+ArrayIPredictorModelInput	Microsoft.ML.Runtime.EntryPoints.CrossValidationBinaryMacro+ArrayIPredictorModelOutput
 Data.TextLoader	Import a dataset from a text file	Microsoft.ML.Runtime.EntryPoints.ImportTextData	TextLoader	Microsoft.ML.Runtime.EntryPoints.ImportTextData+LoaderInput	Microsoft.ML.Runtime.EntryPoints.ImportTextData+Output
+Data.TransformModelArrayConverter	Create and array variable	Microsoft.ML.Runtime.EntryPoints.CrossValidationBinaryMacro	MakeArray	Microsoft.ML.Runtime.EntryPoints.CrossValidationBinaryMacro+ArrayITransformModelInput	Microsoft.ML.Runtime.EntryPoints.CrossValidationBinaryMacro+ArrayITransformModelOutput
```

Comment:
>Create and array variable [](start = 34, length = 25)

So what's the difference between this one and Data.PredictorModelArrayConverter and Data.IDataViewArrayConverter. All of them "Create and array variable"

Reply:
they create array variables of different types, like ITranformModel, IPredictorModel, IDV etc. 

---
In reply to: [191003203](https://github.com/dotnet/machinelearning/pull/212#discussion_r191003203) [](ancestors = 191003203)